google/cloud/aiplatform/training_jobs.py

```diff
@@ -908,9 +908,9 @@ def make_package(self, package_directory: str) -> str:
         shutil.copy(self.script_path, script_out_path)
 
         # Run setup.py to create the source distribution.
-        setup_cmd = self._SETUP_PY_SOURCE_DISTRIBUTION_CMD.format(
-            python_executable=_get_python_executable()
-        ).split()
+        setup_cmd = [
```

Comment:
The key word `{python_executable}` should be removed in the constant above(line 836) because it looks like it goes unused.

Reply:
Ah yes, I was testing on a different repo and didn't copy it over.

Reply:
I fixed it and now addressing some test failures.